src/procedures/ToggleFreezeTransfers.ts

```diff
@@ -17,11 +20,21 @@ export const createToggleFreezeTransfersResolver = (
 };
 
 /**
- * Procedure to transfer security tokens.
+ * Procedure to toggle whether to freeze or unfreeze transfers for a security token
  */
 export class ToggleFreezeTransfers extends Procedure<ToggleFreezeTransfersProcedureArgs> {
   public type = ProcedureType.ToggleFreezeTransfers;
 
+  /**
+   * - Freeze or unfreeze security token transfers
+   *
+   * - Refresh the SDK Security token factory
+   *
+   * Note this procedure will fail if:
+   * - The current wallet address is not the owner
+   * - Trying to freeze a security token that is already frozen
```

Comment:
`- Trying to freeze/unfreeze transfers on a Security Token when they are already frozen/unfrozen`

(This covers the next one as well so delete it)